cmd/minikube/cmd/delete.go

```diff
@@ -107,6 +122,30 @@ func runDelete(cmd *cobra.Command, args []string) {
 	if err := cmdcfg.Unset(pkg_config.MachineProfile); err != nil {
 		exit.WithError("unset minikube profile", err)
 	}
+
+	// Delete the .minikube folder if the flags are set
+	if viper.GetBool(purge) {
+		glog.Infof("Purging the '.minikube' directory located at %s", localpath.MiniPath())
+
+		if viper.GetBool(noPrompt) {
+			glog.Infof("Will not prompt for deletion.")
+		} else {
+			out.T(out.Check, "Are you sure you want to delete the directory located at {{.minikubePath}}? This will delete all your configuration data related to minikube. (Y/N)", out.V{"minikubePath": localpath.MiniPath()})
```

Comment:
maybe check if there are more than 1 profile (default profile) and if there are, tell the user

"this will also delete the configurations for X other profiles "

Reply:
Done. Thanks Medya!